unix_integration/resolver/Cargo.toml

```diff
@@ -110,3 +110,31 @@ kanidmd_testkit = { workspace = true }
 clap = { workspace = true, features = ["derive"] }
 clap_complete = { workspace = true }
 kanidm_build_profiles = { workspace = true }
+
+## Debian packaging
+# We only package unixd itself here as shared libraries must be packaged separarately due to multiarch rules.
+[package.metadata.deb]
+name = "kanidm-unixd"
+maintainer = "James Hodgkinson <james@terminaloutcomes.com>"
+# Can't use $auto depends because the name of libssl3 varies by distro and version
+depends = ["libc6", "tpm-udev", "libssl3 | libssl3t64"]
+section = "network"
+priority = "optional"
+recommends = [
+	"kanidm-pam",
+	"kanidm-nss"
+]
```

Comment:
Can we make these part of `requires`? It's not much use having `kanidm-unixd` without the other integrations IMO, and it makes the install process more usable for humans.